vector/src/main/java/im/vector/app/core/di/ActiveSessionHolder.kt

```diff
@@ -74,6 +75,13 @@ class ActiveSessionHolder @Inject constructor(
         session.callSignalingService().addCallListener(callManager)
         imageManager.onSessionStarted(session)
         guardServiceStarter.start()
+        vectorAnalytics.updateSuperProperties(
+                SuperProperties(
+                        platformCodeName = SuperProperties.PlatformCodeName.EA,
+                        cryptoSDK = SuperProperties.CryptoSDK.Rust,
+                        cryptoSDKVersion = session.cryptoService().getCryptoVersion(applicationContext, false)
+                )
+        )
```

Comment:
`getCryptoVersion` do not need a session to get the info, so maybe this could be done one for all, but it's OK to keep it like this as well.

Rageshake sent before the user is logged in will not contain the SuperProperties, is it OK?

Reply:
I think I still need it, to get a cryptoService? anyhow moved that in a `AutoSuperPropertiesFlowProvider`

Reply:
Fixed now that we have https://github.com/element-hq/element-android/commit/7e41d731f6949191fd9a00c794b7b18f5be293a5